lib/msal-browser/src/config/Configuration.ts

```diff
@@ -51,13 +51,15 @@ export type CacheOptions = {
  * - windowHashTimeout            - sets the timeout for waiting for a response hash in a popup
  * - iframeHashTimeout            - sets the timeout for waiting for a response hash in an iframe
  * - loadFrameTimeout             - maximum time the library should wait for a frame to load
+ * - delayOpenPopup               - Sets whether popup is open as soon as loginPopup is called. By default, this flag is set to false. Set to true for native apps and PWA.
```

Comment:
nit: I would try to rephrase the first sentence to provide both scenarios and make it more generic as it applies to `acquireTokenPopup` as well.  For example: " when set to false, blank popup is opened before anything else happens. When set to true, popup is opened when making the network request"

Reply:
Sounds good! Changed.